charm/charmcraft.yaml

```diff
@@ -44,10 +44,8 @@ config:
   options:
     git_repo:
       type: string
-      description: The repository URL where the DNS records are stored.
+      description: >
+        The repository URL where the DNS records are stored. The username has to be provided.
     git_ssh_key:
```

Comment:
How is the mapping between ssh key and user managed?

Reply:
Same as before ` with TemporaryDirectory() as tmp_dir, Git().custom_environment(GIT_SSH_COMMAND=SSH_EXECUTABLE):`

Reply:
Ok, so we assume that we have one ssh key that is valid for all possible usernames.